### ISSUE TITLE
The Layout Property of Container is incorrectly serialized

### ISSUE THREAD

**Issue by zgcgwin** [extnet/Ext.NET#1776]
The C# code is :
```csharp
Layout = new VBoxLayout()
{
    Align = "center",
    Pack = "center"
}
```

and the render result is:
```js
layout:{align:"center",pack:"center"}
```

expected result is :

```js
layout:{type:"vbox",align:"center",pack:"center"}
```

**Comment by geoffreymcgill** [comment]
Thanks for letting us know about this issue. Somehow we missed catching this during our `7.0.0-preview3` testing. Should be an easy fix. 

We will include the fix in the `7.0.0-preview4` release.

**Comment by AndreyChechel** [comment]
The issue has been fixed and included into `7.0.0-preview4`.
